content/updates/2020-09/1-flat-files-exports-with-labels.md

```diff
@@ -0,0 +1,31 @@
+---
+pim_announcement_img: ./img/Product-export-profile.png
+pim_announcement_alt_img: "Flat files exports with labels"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# Export flat files with labels
+::: meta-data type="New" features="Collaboration, Productivity" available="mid September" link-to-doc="../articles/exports.html"
```

Comment:
For the link to the doc, can you be more precise and linked directly to the anchor where we talk about this new option?